cpp/src/dbscan/vertexdeg/algo.h

```diff
@@ -21,7 +21,6 @@
 #include <common/device_buffer.hpp>
```

Comment:
I think this file also should be renamed as `.cuh` as it ends up calling the distance prim below.

Reply:
Oops, I seem to have missed this. Moved it.